README.md

```diff
@@ -239,10 +246,10 @@ If your extension requires one or more of those plugins to be present, make sure
 The Hide Tribe Bar extension requires that The Events Calendar 3.0 or greater is present. This line is the only one needed to specify that:
 
 ```
-$this->add_required_plugin( 'Tribe__Events__Main', '3.0' );
```

Comment:
Only thing is that the next line is about how The Events Calendar 4.3.3 or higher **is required** — so how can there every be a required plugin under the minimum required version?